app/views/studios/index.html.erb

```diff
@@ -0,0 +1,9 @@
+<h1>All Studios</h1>
+
+<% @studios.each do |studio| %>
+  <u><%= studio.name %> 
+  <%= studio.location %></u> <br>
+  <% studio.studio_movies.each do |movie| %> 
```

Comment:
You could also do `studio.movies` here since a studio already `has_many :movies`. That way, it gets straight into your movie objects and you can use them much easier.